File: app/src/at-dinhtruong/java/asiantech/internship/summer/activityandfragment/InformationActivity.java
package asiantech.internship.summer.activityandfragment;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.TextView;

import asiantech.internship.summer.R;

public class InformationActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_information);
        initInformation();
    }

    private void initInformation() {
        TextView tvEmail = findViewById(R.id.tvEmail);
        TextView tvPassword = findViewById(R.id.tvPassword);
        Intent intent = getIntent();
        tvEmail.setText(intent.getStringExtra(LoginFragment.EMAIL));
        tvPassword.setText(intent.getStringExtra(LoginFragment.PASSWORD));
    }
}

File: app/src/at-baole/java/asiantech/internship/summer/kotlin/activityfragment/InformationActivity.kt
package asiantech.internship.summer.kotlin.activityfragment

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import asiantech.internship.summer.R
import kotlinx.android.synthetic.`at-baole`.kotlin_activity_information.*

class InformationActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.kotlin_activity_information)

        tvEmail.text = intent.getStringExtra(getString(R.string.email))
        tvPassword.text = intent.getStringExtra(getString(R.string.password))
        backActivity()
    }

    private fun backActivity(){
        btnBack.setOnClickListener { onBackPressed() }
    }
}
